core/domain/feature_flag_services_test.py

```diff
@@ -487,8 +487,16 @@ def test_feature_flag_enabled_for_users_in_5_perc_when_increased_to_10_perc(
                     count_feature_flag_enabled_for_5_perc += 1
                     user_ids_for_which_feature_flag_enabled_for_5_perc.add(
                         user_id)
+            # To decide the range in order to avoid the flakiness we are using
```

Comment:
To avoid this test being flaky, we need to determine what the likely range is for the number of users who have the feature flag enabled. This is modelled as a binomial random variable with n = 1000 and p = 0.5. ...

Reply:
Done